### DISCUSSION TITLE
How does replication work?

### DISCUSSION THREAD

**Discussion by lab-at-nohl** [tine20/tine20#7467] [Q&A]
I can see there is a superuser `replicationuser` and also I see code for replication. But how does it work? 

Is it only the DB entries that will be shared between instances? Thus, files and imap have to be transferred separately. 
Is it real time or do I have to trigger sync? If the latter, how?
Is it two instances only or can I have three or more?

Thanks for any insight.

**Comment by pschuele** [comment]
hi Johannes,
I'll write a short article about replication for our new documentation project.
I will let you know when I have something to read ... :smile:
